zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/VariableHandler.java

```diff
@@ -39,7 +40,8 @@ public Class<VariableEntity> getEntityType() {
 
   @Override
   public boolean handlesRecord(final Record<VariableRecordValue> record) {
-    return getHandledValueType().equals(record.getValueType());
+    return getHandledValueType().equals(record.getValueType())
+        && !record.getIntent().name().equals(VariableIntent.MIGRATED.name());
```

Comment:
🔧 let's compare intent objects not names. i.e. `!record.getIntent().equals(VariableIntent.MIGRATED)
🔧 no need to check that the handled value, it's already guaranteed by the other means